src/insert-extension.js

```diff
@@ -59,8 +59,10 @@ function createAmpExtensionScript(win, extension) {
   ampExtensionScript.setAttribute('custom-element', extension);
   ampExtensionScript.setAttribute('data-script', extension);
   const pathStr = win.location.pathname;
+  const useCompiledJs = window.ampTestRuntimeConfig ?
+      window.ampTestRuntimeConfig.useCompiledJs : undefined;
```

Comment:
`window.ampTestRuntimeConfig` is only assigned in testing mode as a global variable. So I guess it doesn't belong to the `win` which is the testing iframe.